ansible/roles/ocp4-workload-iot-next-gen-technologies/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+become_override: false
+repository_archive_url: https://gitlab.com/redhatsummitlabs/next-gen-technologies-at-scale-building-solutions-to-manage-tomorrows-workloads/-/archive/master/next-gen-technologies-at-scale-building-solutions-to-manage-tomorrows-workloads-master.tar.gz
```

Comment:
Can we can pin a version on this ?
(commit ID)

The goal is that if you push something to redhatsummitlabs/next-gen-technologies... it doesn't break deployments.